app/views/sections/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= render partial: 'shared/head', locals: { back: "Back", back_path: user_signed_in? ? user_sections_path(current_user) : splash_path } %>
+
+<div class="container-fluid bg-white p-3">
+
+
+  <div class="group px-3 d-flex flex-column align-items-center">
+     <div class="d-flex justify-content-end align-items-center mb-3">
+    <h4 class="mb-0 mr-2 ms-3">Total Payment:</h4>
+    <h4 class="mb-0 ml-3 mr-5"><b><%=   @total_amount %></b> $</h4>
+  </div>
+    <div class="card w-100">
+      <ul class="list-group list-group-flush">
+        <% @items.each do |item| %>
+          <li class="list-group-item d-flex justify-content-between align-items-center">
+            <div class="d-flex align-items-center">
+              <% if item.icon.attached? %>
+                <%= image_tag item.icon, class: "img-item rounded-circle me-2", style: "width: 40px; height: 40px;" %>
+              <% end %>
+              <h5><%= link_to item.name, user_section_path(current_user, id: item.id), class: "text-decoration-none text-primary mb-0" %></h5>
+            </div>
+            <div class="d-flex flex-column">
+              <span class="fs-4 mb-1"><%= item.payments.map { |item| item.amount }.sum %> $</span>
+              <span class="text-primary date">
+                <b class="text-dark">Date:</b> <%= item.created_at.strftime('%B %d, %Y') %>
+              </span>
+            </div>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+  </div>
+
+  <div class="d-flex justify-content-center mt-3">
+    <%= render partial: 'shared/button', locals: { button_text: "Add New Section", button_path: new_user_section_path(current_user) }, button_class: "btn-lg" %>
+  </div>
+</div>
```

Comment:
##  [OPTIONAL]

- [ ] Kindly note that one of the requirements says that you should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements.should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
KIndly make sure this is fixed
  - Colors
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
 
your design| the temnplate
-----|-----
![image](https://github.com/Daniel-Kigozi/budget-track/assets/65068771/3d8cd334-87f3-4e0f-9df5-e74761973d26) | 
![image](https://github.com/Daniel-Kigozi/budget-track/assets/65068771/0cd30e3d-f9b3-47db-a008-165a1b35be0e)